### PR TITLE
Run ii 94 x v1

### PR DESCRIPTION
**Pull request by garbersc**
updated the tight IDs according to https://twiki.cern.ch/twiki/bin/viewauth/CMS/JetID13TeVRun2017

### CODE REVIEW

File: common/src/JetIds.cxx [hunk 1]
```diff
@@ -92,20 +93,40 @@ bool JetPFID::looseID(const Jet & jet) const{
...
+    
+    if(fabs(jet.eta())>=2.4)
+      return true;
+      
+    if(jet.chargedHadronEnergyFraction()>0
+       && jet.chargedMultiplicity()>0)
+      return true;   
+  }
+  else if(fabs(jet.eta())>2.7 && fabs(jet.eta())<=3
+	  &&jet.neutralEmEnergyFraction()<0.90
```
Comment (raggleton) [inline common/src/JetIds.cxx]: This should be 0.99 according to the twiki page

**Comment by raggleton** [comment]
For the loose ID, since it's no longer an ID in 2017, should we (a) just get rid of it, or (b) add a depreciation printout to the user so they know that the official JetID is either Tight or TightLepVeto?
My worry is that someone transferring their analysis to 2017 will use it and no know that its doing ~nothing.

File: common/src/JetIds.cxx (hunk 1 above)
Comment (garbersc) [reply common/src/JetIds.cxx]: you are right, I corrected this

**Comment by raggleton** [comment]
OK I'll merge this, and then do a separate PR for removing the loose ID

**Pull request merged**